sql/wsrep_xid.cc

```diff
@@ -180,3 +181,35 @@ wsrep::gtid wsrep_get_SE_checkpoint()
 
   return wsrep::gtid(wsrep_xid_uuid(xid),wsrep_xid_seqno(xid));
 }
+
+/*
+  Sort order for XIDs. Wsrep XIDs are sorted according to
+  seqno in ascending order. Non-wsrep XIDs are considered
+  equal among themselves and and greater than with respect
```

Comment:
"equal among themselves and and greater than with respect" ->  "and" repeated twice